go.mod

```diff
@@ -31,6 +34,23 @@ require (
 	github.com/mattn/go-sqlite3 v1.9.0
 	github.com/minio/minio-go v6.0.14+incompatible
 	github.com/peterhellberg/duration v0.0.0-20191119133758-ec6baeebcd10
+	github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d // indirect
```

Comment:
What triggered adding the new dependencies?

Reply:
I'm guessing it's the bazel build.
@DavidSpek can you try removing updates to this file in one commit and see if presubmit still passes (I believe these should be just temporary dependencies fetched when building the protos?

If it does, we'll need to fix the api generation script, so that it doesn't use bazel any more, but it should be outside of this PR.

Reply:
These changes are indeed from running the script. Although it isn't the most reliable, VS code is also lots of these dependencies are not used. I'm removing them now for a new commit to test if things brake.

Reply:
Testing was successful with the changes to this file removed, so I believe the api generation script will need some fixing.